src/tests/Microsoft.PowerFx.Core.Tests/TexlTests.cs

```diff
@@ -363,6 +367,25 @@ public void TexlFunctionTypeSemanticsRenameColumns_Negative(string expression)
             Assert.False(result.IsSuccess);
         }
 
+        [Fact]
+        public void CheckParseSuccess()
+        {
+            var config = new PowerFxConfig();
+            var engine = new Engine(config);
+
+            // var parse = engine.Parse("If(Len(x) = 0, y, x)");
+            var parse = engine.Parse("If(Len(x) = 0, x, y)");
+
+            var r = RecordType.Empty()
+                .Add(new NamedFormulaType("x", FormulaType.String))
+                .Add(new NamedFormulaType("y", FormulaType.UntypedObject));
+
+            var check = engine.Check(parse, r);
+            Assert.True(check.IsSuccess);
+
+            Assert.IsType<StringType>(check.ReturnType);
+        }

```

Comment:
What is this testcase for?

Reply:
This should not be here, some local tests. Will revert prior to getting this PR out of draft